strongbox-user-management/src/main/java/org/carlspring/strongbox/users/service/impl/UserServiceImpl.java

```diff
@@ -44,7 +43,8 @@
     
     @Override
     @Transactional
-    @Cacheable(value = "users", key = "#name", sync = true)
+    //XXX: Cache don't work properly with ORientDB entity proxies
```

Comment:
@Cacheable worked perfectly for me all the time before, but I have to populate cache manually (see UsersConfig for details).

Now we could switch to EhCache 3.2 and follow JSR-107 standard (already did that change in one of my other projects). Usage of EhCache as we have it now in strongbox is obsolete.